src/Symfony/Component/Form/Form.php

```diff
@@ -541,6 +544,10 @@ public function submit($submittedData, $clearMissing = true)
         $viewData = null;
 
         try {
+            if (null !== $this->transformationFailure) {
+                throw $this->transformationFailure;
```

Comment:
we might break things here.. as previously an event could alter the submitted data and validation happened after.

Reply:
the submitted data is already turned to null, there is nothing to alter.

Reply:
oops, you're right missed it :)